README.md

```diff
@@ -1 +1,18 @@
-# tss-ecdsa ![Build Status](https://github.com/novifinancial/tss-ecdsa/workflows/Rust%20CI/badge.svg)
+# tss-ecdsa 
+This repo is a work-in-progress implementation of the UC-Secure threshold ECDSA signing protocol described in https://eprint.iacr.org/2021/060.pdf
+Specifically, we are targeting the three-round presigning protocol (with quadratic overhead for identifying faulty actors).
+
+This codebase is generally intended to be network-agnostic. Programs take messages as input and potentially output some outgoing messages in response. The relaying of these messages is assumed to happen externally. However, a proof-of-concept example of such networking code can be found in examples/network.
+
+## Current State
+### What's Implemented
+Currently, the KeyGen protocol (Figure 5), Auxinfo (Figure 6, minus the key refreshing), and 3 round Presign (Figure 7) are implemented, along with the requisite zero-knowledge proofs. These protocols make use of (where appropriate) an echo-broadcast subprotocol in order to enforce non-equivocation of message contents
```

Comment:
Add a bit of context for what each of these protocols do. "Auxinfo" is particularly confusing naming.

This section would be better if you rephrase to focus on the implemented functionalities, i.e., if you reference the paper only for additional clarity and as a pointer for readers who would like further detail.

Reply:
Also missing period at the end of L9.